### ISSUE TITLE
cls reward

### ISSUE THREAD

**Issue by choasup** [linkangheng/PR1#17]
https://github.com/linkangheng/PR1/blob/accb7cb952f88419c0f66be43b067490036a80a5/src/open_r1/rewards.py#L81

It seems a bug:
pred_logits[range(len(pred_id)), pred_id.long() - 1] = 1.0

**Comment by linkangheng** [comment]
We really appreciate you catching this! The fix has been implemented according to your recommendation. Our tests indicate that this change does not have a significant impact on the final matching results. We will continue to monitor the relevant metrics and will provide an update on any new findings.
